packages/nimble-components/src/menu/styles.ts

```diff
@@ -48,4 +60,13 @@ export const styles = css`
         padding-top: ${smallPadding};
         padding-bottom: ${smallPadding};
     }
-`;
+`.withBehaviors(
```

Comment:
If this pattern now spans multiple controls, should we make it a dedicated theme-aware token? (I haven't looked closely at the pattern to know if that's possible.)

If we do this, we should make sure to consider the existing token `ni-nimble-section-background-color` which currently mentions that it should be used for menu backgrounds. Either we should repurpose it or change its description to distinguish it from a new token.